spec/block_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative "spec_helper"
+describe "Block" do
+  it "creates a block" do
```

Comment:
You should have more tests here, but it looks like you didn't get to it.  

You should also test that it raises an ArgumentError if the number of rooms exceed 5.